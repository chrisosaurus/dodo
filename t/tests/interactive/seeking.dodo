# seek to a small number
b10

# seek to 32 bit int_max
b2147483647

# seek past 32 bit int_max
b4000000000

q
