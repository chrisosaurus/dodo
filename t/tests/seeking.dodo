e/hello there world/

b1
e/ello there world/

b0
e/hello there world/

b18
e/how are you doing today?/

b0
e/hello there world/

b+19
e/ow are/

b-0
e/ow are/

b-1
e/how are/

b-7
e/ world/

b43
e/I am doing well, thank/

