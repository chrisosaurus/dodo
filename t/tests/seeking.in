hello there world
how are you doing today?
I am doing well, thank you
