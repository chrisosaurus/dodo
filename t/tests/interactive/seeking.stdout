dodo [0]: dodo [0]: dodo [10]: dodo [10]: dodo [10]: dodo [2147483647]: dodo [2147483647]: dodo [2147483647]: dodo [4000000000]: dodo [4000000000]: 